netdeploy/remote/deployedNetwork.go

```diff
@@ -455,6 +462,10 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
 	for i := uint64(bootstrappedNet.round); i < fileCfgs.NumRounds; i++ {
 		bootstrappedNet.round++
 		blk, _ := createBlock(src, prev, fileCfgs.RoundTransactionsCount, &bootstrappedNet, params, log)
+		// don't allow the ledger to fall more than 10 rounds behind before adding more
+		for int(bootstrappedNet.round)-int(l.LatestTrackerCommitted()) > 10 {
```

Comment:
is there is a possibility to stuck in infinite loop here if previous commits were discarded and there is no new blocks added => no commits scheduled?